backend/functions/src/api.js

```diff
@@ -14,7 +12,7 @@ const welcome = {
   message: 'If this is your first time? <br /> <b>Enjoy it!</b>',
 };
 
-app.get('/', (req, res) => res.send('Hello World'));
+app.get('/', (req, res) => res.send(process.env.TEST_SECRET || 'Failed'));
```

Comment:
If you wanna try this is the endpoint: https://us-central1-trading-card-manager-mtgx.cloudfunctions.net/api/

Reply:
ta me retornando o hello world